setup/FSharp.SDK/component-groups/Compiler_Redist.wxs

```diff
@@ -124,30 +124,6 @@
         </RegistryKey>
       </Component>
 
-      <Component Id="Compiler_Redist_Microsoft.Build.Conversion.Core.dll" Guid="$(fsharp.guid(Compiler_Redist_Microsoft.Build.Conversion.Core.dll, $(var.LocaleCode)))">
```

Comment:
I wasn't sure. I thought it may have been due to the custom MSBuild build. I'll rollback these changes.